### PR TITLE
fixes #8730 - keep track of components of composite content view versions

### PR DESCRIPTION
**Pull request by jlsherrill**
* adds UI on the CV version details for components
* relaxes restriction on content view versions page to specify content view id
* in support of #8194

### CODE REVIEW

File: db/migrate/20141215213720_track_version_components.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class TrackVersionComponents < ActiveRecord::Migration
+  def up
```
Comment (daviddavis) [inline db/migrate/20141215213720_track_version_components.rb]: I think you can just use `def change` here and drop the down method.

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-components.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<div data-extend-template="layouts/details-nutupane.html">
+
+  <div data-block="header"></div>
+  <div data-block="selection-summary"></div>
+
+  <span class="input-group-addon" data-block="search-filter">
+  </span>
```
Comment (daviddavis) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-components.html]: I think this needs to be removed.

**Comment by daviddavis** [comment]
ACK pending comments and Jenkins

**Comment by daviddavis** [comment]
Actually, I just noticed that the search on the components page doesn't seem to filter at all.

**Comment by jlsherrill** [comment]
@daviddavis    Removed search widget as search isn't supported

**Comment by daviddavis** [comment]
ACK

**Pull request merged**